rest/data/perf.go

```diff
@@ -3,12 +3,12 @@ package data
 import (
 	"context"
 	"fmt"
-	"github.com/evergreen-ci/cedar/units"
-	"github.com/mongodb/grip"
-	"go.mongodb.org/mongo-driver/bson"
 	"net/http"
 	"sort"
 
+	"github.com/evergreen-ci/cedar/units"
+	"github.com/mongodb/grip/message"
+
```

Comment:
this blank line should be omitted.

Reply:
Not sure why goimports keeps adding blank lines.